src/ert/ensemble_evaluator/snapshot.py

```diff
@@ -104,110 +102,175 @@ def convert_iso8601_to_datetime(
     return parse(timestamp)
 
 
+DICT_SEP = "/"
```

Comment:
I think this was only used for the dict flattening code, which is now gone, thus this line can go away.

Reply:
it's gone.